### PR TITLE
Update Format converter docs

### PR DESCRIPTION
**Pull request by marcusolsson**
This PR updates the docs for the Format converter core plugin.

### CODE REVIEW

File: en/How to/Import data.md [hunk 1]
```diff
@@ -32,21 +32,21 @@ If you have data in Roam, it's easy to export and play around with it in Obsidia
 > 4. Click on the vault icon (the tooltip should say "Open another vault") on the bottom left corner in the app, and select "Open folder as vault".
 > Pick the folder you just unzipped to.
 
-> 5. Use our [[markdown format converter]] to convert Roam Research's Markdown format into Obsidian's format.
+> 5. Use our [[vi/Plugin/Markdown format converter]] to convert Roam Research's Markdown format into Obsidian's format.
```
Comment (ericaxu) [inline en/How to/Import data.md:35]: Did you open the `en` folder while working? That would prevent you from linking to pages under other folders

File: en/How to/Import data.md [hunk 2]
```diff
@@ -32,21 +32,21 @@ If you have data in Roam, it's easy to export and play around with it in Obsidia
...
 > It can be used turn `#tag` into Obsidian `[[links]]`.
 > It can also convert `^^highlight^^` into `==highlight==`.
 
 
 ### From Bear
 
-If you're importing from Bear, the [[Markdown format converter]] plugin has an option to convert `::highlight::` to `==highlight==`.
+If you're importing from Bear, the [[vi/Plugin/Markdown format converter]] plugin has an option to convert `::highlight::` to `==highlight==`.
```
Comment (ericaxu) [inline en/How to/Import data.md:42]: ditto

File: en/How to/Import data.md [hunk 3]
```diff
@@ -32,21 +32,21 @@ If you have data in Roam, it's easy to export and play around with it in Obsidia
...
 
 ### Using Zettelkasten
 
 If you've been using the Zettelkasten method to structure your notes, here's something you might want to know:
 
 Links in Obsidian require a full match to the note name. This means that `[[202001010000]]` will not properly link to `[[202001010000 My Note]]`.
 
-In order to fix that, you can use our [[markdown format converter]] to convert your Zettelkasten links into either `[[202001010000 My Note]]` or `[[202001010000 My Note|My Note]]`.
+In order to fix that, you can use our [[vi/Plugin/Markdown format converter]] to convert your Zettelkasten links into either `[[202001010000 My Note]]` or `[[202001010000 My Note|My Note]]`.
```
Comment (ericaxu) [inline en/How to/Import data.md:50]: ditto

File: fr/Plugins/List of plugins.md [hunk 4]
```diff
@@ -21,7 +21,7 @@ Les plugins peuvent être gérés via le bouton "Settings" en bas à gauche. Cer
 - [[Custom CSS]]
 - [[Daily notes]]
 - [[Zettelkasten prefixer]]
-- [[Markdown format converter]]
+- [[fr/Plugins/Markdown format converter]]
```
Comment (ericaxu) [inline fr/Plugins/List of plugins.md:24]: ditto

File: vi/Plugin/List of plugins.md [hunk 5]
```diff
@@ -21,7 +21,7 @@ Các plugin có thể được quản lý thông qua nút "Settings" ở phía d
 1. [[Command palette]]
 1. [[Daily notes]]
 1. [[Zettelkasten prefixer]]
-1. [[Markdown format converter]]
+1. [[vi/Plugin/Markdown format converter]]
```
Comment (ericaxu) [inline vi/Plugin/List of plugins.md:24]: ditto

**Review by ericaxu — CHANGES_REQUESTED** [review]

**Comment by ericaxu** [comment]
Just some path issues, otherwise look good to me.

File: en/How to/Import data.md (hunk 1 above)
Comment (marcusolsson) [reply en/How to/Import data.md:35]:
```suggestion
> 5. Use our [[Format converter]] to convert Roam Research's Markdown format into Obsidian's format.
```

File: en/How to/Import data.md (hunk 2 above)
Comment (marcusolsson) [reply en/How to/Import data.md:42]:
```suggestion
If you're importing from Bear, the [[Format converter]] plugin has an option to convert `::highlight::` to `==highlight==`.
```

File: en/How to/Import data.md (hunk 3 above)
Comment (marcusolsson) [reply en/How to/Import data.md:50]:
```suggestion
In order to fix that, you can use our [[Format converter]] to convert your Zettelkasten links into either `[[202001010000 My Note]]` or `[[202001010000 My Note|My Note]]`.
```

File: vi/Plugin/List of plugins.md (hunk 5 above)
Comment (marcusolsson) [reply vi/Plugin/List of plugins.md:24]:
```suggestion
1. [[Format converter]]
```

**Review by ericaxu — APPROVED** [review]

**Pull request merged**